lib/WP_Auth0_LoginManager.php

```diff
@@ -91,14 +91,12 @@ public function logout() {
     $auto_login = absint( $this->a0_options->get( 'auto_login' ) );
 
     if ( $slo && isset( $_REQUEST['SLO'] ) ) {
-      $redirect_to = $_REQUEST['redirect_to'];
-      wp_redirect( $redirect_to );
+      wp_redirect( $_REQUEST['redirect_to'] );
```

Comment:
does this wp_redirect method validate the redirect_to in some way?

Reply:
@cocojoe - Uses wp_sanitize_redirect()